File: exercicios/R/1008/1008.r
input = file('stdin','r')

cod = as.integer(readLines(input, n=1))
horas = as.integer(readLines(input, n=1))
valor = as.double(readLines(input, n=1))

salario = valor * horas

write(sprintf("NUMBER = %d",cod),'')
write(sprintf("SALARY = U$ %.2f",salario),'')

File: exercicios/JavaScript/1008/1008.js
var input = require('fs').readFileSync('/dev/stdin','utf8');
var lines = input.split('\n');

var cod = parseInt(lines.shift());
var qt = parseInt(lines.shift());
var vl = parseFloat(lines.shift());

var salario = qt * vl;

console.log("NUMBER = " + cod);
console.log("SALARY = U$ " + salario.toFixed(2));